openwisp_controller/config/base/vpn.py

```diff
@@ -18,6 +18,17 @@ class AbstractVpn(ShareableOrgMixin, BaseConfig):
     Abstract VPN model
     """
 
+    default_dh = (
```

Comment:
let's rename to: `_placeholder_dh` because 1 it's not a default but it's a placeholder and 2 the leading underscore flags it as private.